src/components/styles/book.css

```diff
@@ -1,19 +1,16 @@
 .book-card {
-  border: 1px solid #e8e8e8;
-  margin-top: 40px;
-  padding: 2rem;
-  background-color: #fff;
-  border-radius: 4px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 }
 
 .book-info {
   display: flex;
   flex-direction: column;
-  width: 100%;
+  width: 40%;
 }
 
 .book-title {
-  padding: 7px 0;
   margin-top: 0.188rem;
   font-size: 1.375rem;
   font-weight: 600;
```

Comment:
- [ ] The styling has to be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please add the missing vertical bars ✍🏻 

![18](https://user-images.githubusercontent.com/79058364/194730268-939022e6-22b3-4878-bc8c-29c6900dc105.png)

Reply:
@BenMukebo  done

Reply:
Great 👍🏻 @AlaaShaheen2